pkg/daemon/update.go

```diff
@@ -1809,6 +1828,41 @@ func (dn *Daemon) atomicallyWriteSSHKey(keys string) error {
 	return nil
 }
 
+// Set a given PasswdUser's Password Hash
+func (dn *Daemon) SetPasswordHash(newUsers []ign3types.PasswdUser) error {
+	// confirm that user exits
+	if len(newUsers) == 0 {
+		return nil
+	}
+
+	// mock function to mock the nodes existence  (it shouldnt run if no node exists)
```

Comment:
don't understand the need of this comment. Are we planning to mock this?

Reply:
this was taken from the update ssh keys function- I believe that chunk of code was to mock the nodes existence to ensure it runs if a node exists